src/OrchardCore.Modules/OrchardCore.Autoroute/Liquid/ContentAutorouteLiquidTemplateEventHandler.cs

```diff
@@ -24,7 +24,7 @@ public Task RenderingAsync(TemplateContext context)
             {
                 return new LiquidPropertyAccessor(async alias =>
                 {
-                    if (!alias.StartsWith("/"))
+                    if (!alias.StartsWith('/'))
```

Comment:
Use `Path.AltDirectorySeparatorChar` everywhere :)

Reply:
Too long ;)

Here it's more a route pattern segment separator, in the aspnetcore source, in `UriHelper.cs` just saw `private const char ForwardSlash = '/'`. But e.g in `PathString.cs` it is used as is `/` and seems to be the trend.

Reply:
We can do it at least like this instead
```csharp
internal static readonly char SegmentSeparator = '/';
```

Reply:
Yes but there would be so many place to change ;)

And using '/' seems to be the trend in aspnetcore code, e.g

    var segments = pageRouteMetadata.PageRoute.Split('/');
    for (var i = 0; i < segments.Length; i++)
    {
        segments[i] = model.RouteParameterTransformer.TransformOutbound(segments[i]);
    }

    var transformedPageRoute = string.Join("/", segments);

Reply:
Also, because i use `/` as a char and also as a string, i would need something like this.

        internal const char SeparatorChar = '/';
        internal const string SeparatorString = "/";

                    if (!alias.StartsWith(SeparatorChar))
                    {
                        alias = SeparatorString + alias;
                    }

I think it's more readable to hardcode the conventional `/`.

                    if (!alias.StartsWith('/'))
                    {
                        alias = "/" + alias;
                    }

So, can you remove your requested change? But only if you agree ;)